src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form form--big" data-qa="big">
+
+      <label
+      for="form--big__input"
+      class="form__label form__label--big">
+
+        <img src="./images/Search.svg" alt="search">
+
+      </label>
+
+      <input
+      class="form__input form__input--big"
+      data-qa="keypress"
+      id="form--big__input"
+      placeholder="Try “Los Angeles”"
+      type="text">
```

Comment:
Need to fix code style everywhere
```suggestion
      <input
        class="form__input form__input--big"
        data-qa="keypress"
        id="form--big__input"
        placeholder="Try “Los Angeles”"
        type="text"
      >
```